src/object/srv_obj_migrate.c

```diff
@@ -189,8 +189,11 @@ obj_tree_create(daos_handle_t toh, void *key, size_t key_size,
 
 	*rootp = tmp_root;
 out:
-	if (rc < 0)
-		dbtree_delete(toh, BTR_PROBE_EQ, &key_iov, NULL);
+	if (rc < 0) {
+		rc2 = dbtree_delete(toh, BTR_PROBE_EQ, &key_iov, NULL);
+		if (rc2)
+			D_WARN("failed to delete \"%s\": "DF_RC"\n", (char *)key, DP_RC(rc2));
```

Comment:
This is not correct.  It needs to be DF_KEY and DP_KEY.  This has a couple of effects

1. On release builds, it doesn't print any data (which may be private)
2. It only prints a portion of a long key
3. It prints an integer key as an integer
4. It doesn't print non-ascii characters

Reply:
I just changed it to, 
D_WARN("failed to delete "DF_KEY": "DF_RC"\n", DP_KEY(&key_iov), DP_RC(rc2));

Thank you very much!